app/gui_api.py

```diff
@@ -3,10 +3,20 @@
 
 class GuiApi:
 
-    def __init__(self, auth_svc, plugins):
-        self.auth_svc = auth_svc
+    def __init__(self, services, plugins):
+        self.services = services
+        self.caldera_config = services.get('utility_svc').get_config()
+        self.auth_svc = services.get('auth_svc')
```

Comment:
let's remove this single service from being called out in the initialization. find it's usage and just do: services.get('auth_svc').